README.md

```diff
@@ -65,6 +65,10 @@ $ swift package format --targets AirbnbSwiftFormatTool
 $ swift package format --swift-version 5.3
 ```
 
+The package plugin returns a non-zero exit code if there is a lint failure that requires attention.
+ - In `--lint` mode, any lint failure from any tool will result in a non-zero exit code.
+ - In standard autocorrect mode without `--lint`, only failures from SwiftLint lint-only rules will result in a non-zero exit code.
```

Comment:
@calda do you know if there is the possibility of a situation where there is an autocorrectable rule that can't be autocorrected in all cases (and thus would continue to result in a non-zero exit code after autocorrection)?

Reply:
Not sure. My guess is no but I don't know.